helix-core/src/test/java/org/apache/helix/integration/task/TestEnqueueJobs.java

```diff
@@ -41,13 +47,39 @@
   public void beforeClass() throws Exception {
     setSingleTestEnvironment();
     super.beforeClass();
+    //WorkflowConfig.disableJobPurge();
+  }
+
+  @AfterClass
+  @Override
+  public void afterClass() throws Exception {
```

Comment:
I did a quick test, I think you can just remove it and the super.afterClass() will still be called, since you don't have any method with the same name to override it in that case.

Reply:
These @beforeClass and @afterClass is due to the old way using WorkflowConfig.disableJobPurge(). Should be removed.